routes/api.php

```diff
@@ -31,6 +31,8 @@
             'items.aggregations'
         );
         Route::get('items/{id}', [V1ItemController::class, 'detail'])->name('items.show');
+        Route::post('items/{id}/increment-view-count', [V1ItemController::class, 'incrementViewCount'])
+            ->name('items.increment-view-count');
```

Comment:
`/increment-view-count` is very clear but maybe a bit SOAPy?

```suggestion
        Route::post('items/{id}/views', [V1ItemController::class, 'incrementViewCount'])
            ->name('items.increment-view-count');
```

(It really is just an idea, take it or leave it)